windows/src/engine/keyman32/kmhook_getmessage.cpp

```diff
@@ -412,6 +419,24 @@ void ProcessWMKeymanControlInternal(HWND hwnd, WPARAM wParam, LPARAM lParam)
 	}
 }
 
+void
+ProcessWMKeymanControl(WPARAM wParam, LPARAM lParam) {
+  WORD wAtom = HIWORD(lParam);
```

Comment:
This will work fine but technically the `wAtom` is only available with `KMC_PROFILECHANGED`, so we should leave the declaration here and move the assignment to just above line 433 (`GetAtomName`) for clarity and to avoid misleading future devs...

Reply:
So that is sort of what I had before. I didn't actually get this statement I just did it anyway. 

>  Can we move these outside the switch statement as their scope is rubbery otherwise... 

I am not sure why it all can't just be inside the KMC_PROFILECHANGED as it is the only scope it makes sense.

Reply:
@rc-swag Shall I test this PR in Windows 10 OS and post my comment? (since I have only Windows 10 OS in my machine) Your Suggestion, Please.

Reply:
> test this PR in Windows 10 OS and post my comment? (since I have only Windows 10 OS in my machine) Your Suggestion, Please.

@bharanidharanj  unfortunately this problem only occurred in windows 11. So 
@MakaraSok  do you have a win 11 VM?
Otherwise @mcdurdin  I can I test ( I already have my own PR) on this occasion?

Reply:
> Otherwise @mcdurdin can I test ( I already have my own PR) on this occasion?

Sure. There's no real restriction on who does tests -- Makara and Bharani are typically setup to run tests faster and it's good to get a fresh set of eyes on it but I'm all for reducing friction!